src/Pact/Parse.hs

```diff
@@ -88,7 +88,8 @@ expr = do
    (EList <$> parens (many expr) <*> pure Nothing <*> inf <?> "sexp")
    <|>
    do
-     is <- brackets (many expr) <?> "list literal"
+     is <- (TF.try (brackets (many expr) <?> "space-delimited list literal") <|>
+                   (brackets ((`sepBy` comma) expr) <?> "comma-delimited list literal"))
```

Comment:
You might want to remove the redundant outer parens of this expression, and/or change the use of `sepBy` to ``expr `sepBy` comma``, but otherwise LGTM.